### PR TITLE
Update benchmark dependencies

### PR DESCRIPTION
**Pull request by davidbgk**
* Replaces `apistar` with `starlette`.
* Updates `uvicorn` example

Note that `japronto`, `aiohttp` and `responder` are not anymore working, I'm not sure if we want to keep those or replace with more recent ones?

### CODE REVIEW

**Comment by yohanboniface** [comment]
> Note that japronto, aiohttp and responder are not anymore working, I'm not sure if we want to keep those or replace with more recent ones?

If they are still maintained and we can make the latest version work easily, we keep them, otherwise let's drop ?

**Comment by trollfot** [comment]
aiohttp is still a landmark in the asyncio frameworks. Japronto is a work-in progress early-adopters only kind of framework. Responder is not very popular but still alive.

**Comment by davidbgk** [comment]
For the record, see also https://github.com/TechEmpower/FrameworkBenchmarks/ => https://www.techempower.com/benchmarks/ (for Python frameworks: https://www.techempower.com/benchmarks/#section=data-r18&hw=ph&test=json&l=zijzen-f )

File: benchmarks/starlette/run.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+exec gunicorn app:app --workers $WORKERS --threads $WORKERS --config gunicorn_conf.py
```
Comment (yohanboniface) [inline benchmarks/starlette/run.sh]: If worker is uvicorn, why don't we simply use the uvicorn executable ?

Comment (davidbgk) [reply benchmarks/starlette/run.sh]: I followed the documentation: https://www.starlette.io/#performance maybe there is a better/simpler way?

Comment (yohanboniface) [reply benchmarks/starlette/run.sh]: OK, I'd say let's follow the doc :)

**Pull request merged**